client/a8c-for-agencies/components/sidebar/index.tsx

```diff
@@ -87,11 +87,10 @@ const A4ASidebar = ( {
 				<ul>
 					{ withGetHelpLink && (
 						<SidebarNavigatorMenuItem
-							isExternalLink
 							title={ translate( 'Get help', {
 								comment: 'A4A sidebar navigation item',
 							} ) }
-							link="https://agencies.automattic.com/support"
+							link="https://agencies.automattic.com/overview#contact-support"
```

Comment:
```suggestion
							link="/overview#contact-support"
```

Reply:
Same idea here.

Reply:
The contact form doesn't currently open when navigating to the page within the app, using the full URL here triggers a fresh page load which opens the contact form.

I realize it is a bit lazy, we could instead update the overview page to support opening the form on in-app navigation, or better yet add a app-wide hook to open the support form on any page.

The priority of implementing that full solution I will leave up to you! Happy to update the PR if necessary.

Reply:
Let's go ahead and implement the full solution. I think it is worth it. The full-page reload feels confusing.